tst/EnergyPlus/unit/HighTempRadiantSystem.unit.cc

```diff
@@ -137,5 +141,35 @@ namespace EnergyPlus {
 
 	}
 
+	TEST_F( EnergyPlusFixture, HighTempRadiantSystemTest_SizeHighTempRadiantSystemScalableFlagSetTest )
+	{
+		int RadSysNum;
+		int SizingTypesNum;
+		
+		HighTempRadiantSystem::clear_state();
```

Comment:
@mjwitte I'll look at these and add as appropriate (DataScalableCapSizingON = false).